owncloudApp/src/main/java/com/owncloud/android/operations/RefreshFolderOperation.java

```diff
@@ -122,7 +122,13 @@ public RefreshFolderOperation(OCFile folder,
         OwnCloudVersion serverVersion = null;
 
         // get 'fresh data' from the database
-        mLocalFolder = getStorageManager().getFileByPath(mLocalFolder.getRemotePath());
+        OCFile tempFile = getStorageManager().getFileByPath(mLocalFolder.getRemotePath());
+
+        if (tempFile != null) {
+            mLocalFolder = tempFile;
+        } else {
+            Timber.e("Error, file not received because mLocalFolder is null");
```

Comment:
Let's try to make the log a little bit more useful just in case we reproduce the issue in the future. What do u think? 
```suggestion
            Timber.w("File with path: " + mLocalFolder.getRemotePath() + " and account " +  mAccount.name + " could not be retrieved from database for account : " + getStorageManager().getAccount().name + " Let's try to synchronize it anyway");
```

Reply:
Excellent! Thank u for the suggestion 😉